hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties

```diff
@@ -168,7 +168,7 @@ tokens.nfts.maxMetadataBytes=100
 tokens.nfts.maxBatchSizeBurn=10
 tokens.nfts.maxBatchSizeWipe=10
 tokens.nfts.maxBatchSizeMint=10
-tokens.nfts.maxAllowedMints=10_000_000
+tokens.nfts.maxAllowedMints=20_000_000
```

Comment:
Q: Does this need to be replicated in modularized `application.properties` or was this already done?

Reply:
we did it in the configuration file that you can see in the pr
@ConfigProperty(value = "nfts.maxAllowedMints", defaultValue = "20000000") @NetworkProperty